fiftyone/operators/executor.py

```diff
@@ -428,6 +430,10 @@ def trigger(self, operator_name, params=None):
         Args:
             operator_name: the name of the operator
             params (None): a dictionary of parameters for the operator
+
+        Returns:
+            a :class:`fiftyone.operators.message.GeneratedMessage` containing
+            the result of the invocation
```

Comment:
```suggestion
            the name and params of an operator to execute in the app
```

Reply:
Here I'm documenting the return argument, which does look to be a `GeneratedMessage` based on the implementation

Reply:
GeneratedMessage is the container- it’s like saying it returns a tesponse

Reply:
Sure but that's what `Returns` is for. You can click on `GeneratedMessage` and see its properties if you want

Reply:
Just like response it has generic properties like body. It’s fine I can tweak later.